rpcserver.go

```diff
@@ -7926,11 +7926,64 @@ func (r *rpcServer) AddInvoice(ctx context.Context,
 	}
 
 	return &tchrpc.AddInvoiceResponse{
-		AcceptedBuyQuote: acceptedQuote,
+		// TODO(george): For now we just return the expensive quote.
+		AcceptedBuyQuote: expensiveQuote,
 		InvoiceResult:    invoiceResp,
 	}, nil
 }
 
+func (r *rpcServer) AcquireBuyOrder(ctx context.Context,
```

Comment:
Missing `godoc` comment.

Reply:
Also does this _need_ to live in the RPC server? If not, then just like the refactor earlier, we can move it into the `rfq` package. This way we can add tests w/o needing to mock out the various RPC sever deps.

Reply:
Gotcha, I think we can at least make it private then? Otherwise it looks like an RPC we're implementing vs a helper method.